test/powershell/Host/ConsoleHost.Tests.ps1

```diff
@@ -244,17 +244,17 @@ Describe "ConsoleHost unit tests" -tags "Feature" {
 '@
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-windows`
Tests for Get-ComputerInfo: Ensure Type returned.Verify type returned by Get-ComputerInfo
```powershell
Expected the value to have type [Microsoft.PowerShell.Commands.ComputerInfo] or any of its subtypes, but got @{BiosCaption=Intel(R) Xeon(R) CPU E5-2673 v4 @ 2.30GHz} with type [PSCustomObject].
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\Get-ComputerInfo.Tests.ps1: line 1011
1011:             $computerInfo | Should -BeOfType 'Microsoft.PowerShell.Commands.ComputerInfo'
```